app/src/main/java/com/vero/tasky/agenda/domain/model/NetworkPhoto.kt

```diff
@@ -1,6 +1,6 @@
 package com.vero.tasky.agenda.domain.model
 
-data class Photo (
+data class NetworkPhoto (
```

Comment:
I like Photo (or AgendaPhoto) way more
Because you'd need to separate Local from Network/Remote photo

So you can have a sealed class as a parent (like with AgendaItem)

Example:
```
sealed class AgendaPhoto(val location: String) {
    data class Local(val localUri: String) : AgendaPhoto(localUri)
    data class Remote(val remoteUrl: String, val key: String) : AgendaPhoto(remoteUrl)
}
```